app/storybook/documentation/ui-kit/hooks/QueryHooks.mdx

```diff
@@ -0,0 +1,61 @@
+import { Canvas, ArgTypes, Meta, Story, Source, Description, Primary, Controls, Stories } from '@storybook/blocks'
+
+<Meta title="HOOKS/Query Hooks" /> 
```

Comment:
Let's put it into [Customization](https://ui-488jmcpzw-propeldata.vercel.app/?path=%2Fdocs%2Fcustomization-theming--overview) section.